css-three-column-layout/styles.css

```diff
@@ -0,0 +1,57 @@
+.container {
+  /* max-width: 800px; */
+  margin: auto;
+
+  /* justify-content: center; */
+}
+
+body {
+  background-color: #1b96d9;
+  font-family: Lato, sans-serif;
+  font-style: normal;
+  text-align: center;
+  color: white;
+}
+
+.row {
+  display: flex;
+}
+
+.header {
+  justify-content: center;
+  padding-top: 40px;
+}
+
+.column-third {
+  flex-basis: calc(100% / 3);
+  margin: 30px;
```

Comment:
Don't pollute your layout classes. The only styling on `column` classes should be `flex-basis`. If you need this, you should apply it to another class you can add in addition to the column class.